docs/http.rst

```diff
@@ -133,11 +133,14 @@ Calls
 Send Notification
 ~~~~~~~~~~~~~~~~~
 
-Send a notification to the given endpoint identified by it's `token`.
+Send a notification to the given endpoint identified by it's `push_endpoint`.
```

Comment:
yeah, there's a bit of a difference between pointing out mistakes and, well, Nazism. So, yeah, feel free to point out my mistakes.